.circleci/config.yml

```diff
@@ -107,7 +107,7 @@ executors:
     parameters:
       resource_class:
         type: string
-        default: large
+        default: xlarge
```

Comment:
For posterity, plan is to revert this back to `large` when FXA-7958 lands?

Reply:
I'm unsure of the plan here, but I believe we'd like to get back to a large instance. I don't believe this PR is the only one causing us to bump to xlarge.